tests/test_crypto_com_app.py

```diff
@@ -0,0 +1,251 @@
+        assert in_transaction.spot_price == "1235.649212"
+        assert RP2Decimal(in_transaction.crypto_in) == RP2Decimal("0.025")
+        assert in_transaction.fiat_in_with_fee == "30.8912303"
+        assert in_transaction.fiat_in_no_fee == "30.8912303"
+        assert in_transaction.crypto_fee == "0"
+
+    def test_handle_intra_transaction(self) -> None:
+        plugin = InputPlugin(
+            account_holder="tester",
+            in_csv_file="input/test_crypto_com_app.csv",
+            native_fiat="USD",
+        )
+
+        transaction = CryptoComAppTransaction(
+            raw_data="11/21/2022 22:08,Withdraw ETH (ERC20),ETH,-0.3877,USD,421.5213882,USD,421.5213882,"
+            "crypto_withdrawal,0x1646a7c9f57f5f543c498f5ebff14dab16f9c9b09dbfb50405622564b8c62762",
+            time=plugin.format_time("11/21/2022 22:08"),
+            description="Withdraw ETH (ERC20)",
+            currency="ETH",
+            amount="-0.3877",
+            native_currency="USD",
+            native_amount="421.5213882",
+            native_amount_usd="421.5213882",
+            transaction_kind="crypto_withdrawal",
+        )
+
+        result = plugin.handle_intra_type_transaction(transaction)
+
+        assert isinstance(result, IntraTransaction)
+        assert result.asset == "ETH"
+        assert result.timestamp == "2022-11-21 22:08:00+0000"
+        assert result.from_exchange == "Crypto.com App"
+        assert result.to_exchange == "tester_External"
+        assert result.from_holder == "tester"
+        assert result.to_holder == "tester"
+        assert math.isclose(float(result.crypto_sent), float(transaction.amount), rel_tol=1e-9)
+        assert RP2Decimal(result.crypto_sent) == RP2Decimal("0.3877")
+        assert RP2Decimal(result.crypto_received) == RP2Decimal("0.3877")
+
+    def test_handle_in_transaction(self) -> None:
+        plugin = InputPlugin(
+            account_holder="tester",
+            in_csv_file="input/test_crypto_com_app.csv",
+            native_fiat="USD",
+        )
+
+        transaction = CryptoComAppTransaction(
+            raw_data="3/31/2022 0:03,Sign-up Bonus Unlocked,CRO,61.73662329,USD,25,USD,25,25,referral_gift",
+            time=plugin.format_time("3/31/2022 0:03"),
+            description="Sign-up Bonus Unlocked",
+            currency="CRO",
+            amount="61.73662329",
+            native_currency="USD",
+            native_amount="25",
+            native_amount_usd="25",
+            transaction_kind="referral_gift",
+        )
+
+        result = plugin.handle_in_type_transaction(transaction)
+
+        assert isinstance(result, InTransaction)
+        assert result.asset == "CRO"
+        assert result.timestamp == "2022-03-31 00:03:00+0000"
+        assert result.transaction_type.lower() == Keyword.INCOME.value.lower()
+        assert math.isclose(float(result.crypto_in), float(transaction.amount), rel_tol=1e-9)
```

Comment:
After switching to RP2Decimal, use is_equal_within_precision instead of isclose (seej https://github.com/eprbell/rp2/blob/main/src/rp2/out_transaction.py#L99). Same thing in the rest of the file.